.github/workflows/stale.yml

```diff
@@ -16,3 +16,5 @@ jobs:
         stale-issue-label: 'stale'
         days-before-stale: 30
         days-before-close: 5
+        start-date: '2018-06-01T00:00:00+00:00'
+        exempt-issue-labels: 'priority/critical,priority/important-longterm,priority/important-soon,priority/weave,good first issue'
```

Comment:
```suggestion
        exempt-issue-labels: 'priority/critical,priority/important-longterm,priority/important-soon,priority/weave,good first issue,pinned'
```

Reply:
oh geez github doesn't expose pinnedness through the REST API, how painful

Reply:
it might 🤷‍♀️  but this thing doesn't appear to be looking for it